src/bundle/Resources/public/scss/_field-group.scss

```diff
@@ -73,12 +34,49 @@
         font-size: calculateRem(18px);
     }
 
+    &__toggler {
+        position: relative;
+        display: block;
+        width: 100%;
+        margin-bottom: calculateRem(10px);
+        padding: calculateRem(5px) 0;
+        font-size: calculateRem(17px);
+        font-weight: 400;
+        color: $ez-color-secondary;
+        border-bottom: calculateRem(1px) solid;
+
+        &:hover {
+            color: $ez-color-secondary;
+            text-decoration: none;
+        }
+
+        &:after {
+            content: '';
+            position: absolute;
+            top: calculateRem(20px);
+            right: calculateRem(5px);
+            display: inline-block;
+            width: calculateRem(7px);
+            height: calculateRem(7px);
+            border-right: calculateRem(2px) solid $ez-color-secondary;
+            border-bottom: calculateRem(2px) solid $ez-color-secondary;
+            transition: all 0.2s $ez-admin-transition;
```

Comment:
I think we try to standarize transitions lately and we add 0.3 as transition time, but maybe it would be better to ask Inaki.